### PR TITLE
Add sql scripts and diagrams

### PR DESCRIPTION
**Pull request by VRublevski**

### CODE REVIEW

**Review by papahmsolo — CHANGES_REQUESTED** [review]
1. Estimation
2. ER diagram
3. Status table, Format table + table fields
4. UUID PK
5. DB scripts: create a separate schema, don't delete the DB if it exists, employ `IF NOT EXISTS`

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Clone the repository. 
+2. Install PostgresSQL database version 10.15 or higher. You can find the latest version by visiting:
+
+   https://www.postgresql.org/download/
+3. Open your command line tool and switch you working directory to the `scripts` folder of this repository. Then run
+
+ ```
+ psql -f schema.sql
+ ```
```
Comment (papahmsolo) [inline README.md:17]: Add the user parameter to this command, I'll only have the admin (`postgres`) user after installation

File: scripts/schema.sql [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	target_format converter.video_format NOT NULL,
+	bitrate integer,
+	ratio integer,
+	created timestamp without time zone NOT NULL,
+	updated timestamp without time zone NOT NULL,
+	request_id uuid NOT NULL,
+	CONSTRAINT parameters_request__fk FOREIGN KEY (request_id)
+		REFERENCES converter.request (id)
+		ON DELETE CASCADE
+);
```
Comment (papahmsolo) [inline scripts/schema.sql:78]: New line?

**Review by papahmsolo — CHANGES_REQUESTED** [review]
This looks very good, Vlad!

- Let's use the `text` type instead of `varchar` in `schema.sql`, otherwise, it looks good.
- You'll need more SQL queries for your services, e.g. create the user. Add all the missing ones.

File: scripts/schema.sql (hunk 2 above)
Comment (VRublevski) [reply scripts/schema.sql:80]: @papahmsolo I don't know how she got here.

**Review by papahmsolo — APPROVED** [review]
Me like!

**Pull request merged**